app/src/main/java/com/example/anidb/data/repository/source/AnimeDataSource.kt

```diff
@@ -0,0 +1,16 @@
+package com.example.anidb.data.repository.source
+
+import com.example.anidb.data.model.Anime
+import com.example.anidb.data.repository.source.remote.OnResultListener
+
+interface AnimeDataSource {
+    interface Local
+
+    interface Remote {
+        fun getAnimePopular(limit: Int, page: Int, listener: OnResultListener<MutableList<Anime>>)
```

Comment:
Data trả về từ Api là cố định, dùng List thay vì MutableList em nha, sau này khi cần sửa đổi ta sẽ chuyển sang mutable sau
```suggestion
        fun getAnimePopular(limit: Int, page: Int, listener: OnResultListener<List<Anime>>)
```